packages/language-service/ivy/test/gettcb_spec.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {absoluteFrom} from '@angular/compiler-cli/src/ngtsc/file_system';
+import {initMockFileSystem, TestFile} from '@angular/compiler-cli/src/ngtsc/file_system/testing';
+
+import {extractCursorInfo, LanguageServiceTestEnvironment} from './env';
+
+describe('get typecheck block', () => {
+  let env: LanguageServiceTestEnvironment;
+
+  beforeEach(() => {
+    initMockFileSystem('Native');
+  });
+
+  it('should find the typecheck block for an inline template', () => {
+    const {text, cursor} = extractCursorInfo(`
+      import {Component} from '@angular/core';
+
+      @Component({
+        template: '<div>{{ my¦Prop }}</div>',
+      })
+      export class AppCmp {
+        myProp!: string;
+      }`);
+    const appFi = absoluteFrom('/app.ts');
+    createModuleWithDeclarations([{name: appFi, contents: text}]);
+
+    env.expectNoSourceDiagnostics();
+    const result = env.ngLS.getTcb(appFi, cursor);
+    // FIXME: why is fail not String -> Bottom?
+    if ('error' in result) {
+      fail('Expeced a valid TCB response');
+      return;
+    }
```

Comment:
```suggestion
    if ('error' in result) {
      fail('Expeced a valid TCB response');
      return;
    }
```
Can we just remove this and allow the other expectations to fail?

Reply:
We could but I feel that this is a bit clearer when the test fails and avoids a non-null assertion. If you would prefer the removal of this branch let me know.